tests/lowering/misc/test_select.py

```diff
@@ -0,0 +1,40 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class SelectModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input):
+        return torch.select(input, 3, 0)
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
```

Comment:
Maybe we can add select dim and index as test parameters, to enhance the strength of the tests

Reply:
done. refer to 08b155c5c66a408a1fe6ffc6109be103747cc444. also add more test parameters.